common/api/core-frontend.api.md

```diff
@@ -1916,10 +1916,11 @@ export class Cluster<T extends Marker> {
     constructor(markers: T[]);
     // (undocumented)
     clusterMarker?: Marker;
+    getClusterLocation(): Point3d;
     // (undocumented)
     readonly markers: T[];
     // (undocumented)
-    readonly rect: ViewRect;

```

Comment:
I just noticed you have introduced a breaking change to a public API.

Reply:
Yes, the clustering algorithm no longer needs to maintain a rect for the clusters. I don't know what to do about this breaking change. Is it better to leave the rect property in place and document it as unused?

Reply:
You can either sneak it in as a last-minute breaking change for 3.0, in which case you must document it in NextVersion.md; or you can mark it `@deprecated` and remove it in 4.0.
Once 3.0 is officially released deprecating is the only option.
docs/learning/guidelines/release-tags-guidelines.md.

Reply:
I documented it in NextVersion.md, please let me know if it needs any edits.